cordova-js-src/splashscreen.js

```diff
@@ -148,29 +152,27 @@ function init(config, manifest) {
     extendedSplashProgress.classList.add('win-medium');
     extendedSplashProgress.classList.add('win-ring');
 
+    extendedSplashImage.style.maxWidth = "100%";
+    extendedSplashImage.style.maxHeight = "100%";
+    extendedSplashImage.src = splashImageSrc;
+
     if (isWp81 || isWp10) {
-        extendedSplashImage.style.maxWidth = "100%";
-        extendedSplashImage.style.maxHeight = "100%";
-        extendedSplashImage.src = splashImageSrc;
         // center horizontally
         extendedSplashImage.style.margin = "0 auto";
-        extendedSplashImage.style.display = "block";
+        extendedSplashImage.style.display = "inline-block";
```

Comment:
Can we move this to CSS using the same technique as for `extendedSplashProgress` (additional css classes based on whether we're running on desktop/phone)?